src/app/components/work-item-quick-add/work-item-quick-add.component.less

```diff
@@ -40,10 +40,12 @@
     &-story {
       padding-right: 70px;
       padding-left: 42px;
-      textarea[class='f8-quickadd-desc'] {
-        max-height: 100px;
-        height: 26px !important;
-        padding-top: 0;
+      textarea.f8-quickadd-desc {
+        bottom: -25px;
+        max-height: 200px;
+        max-width: 407px;
+        position: absolute;
```

Comment:
why position absolute?

Reply:
We want the description field to grow in vertically upward direction. That's why it's position has to absolute. Without this, the textarea will grow vertically downward.